.github/ISSUE_TEMPLATE/package.md

```diff
@@ -0,0 +1,28 @@
+---
+name: Package Request
+about: Create a proposal for a new Stencil Community package
+title: ""
+labels: package proposal
+assignees: ""
+---
+
+<!--
+Thank you for adding a new package proposal!
+Please fill out the following template.
+-->
+
+## Package Request
+
+<!--
+Propose a human-readable name for the package as well as its package name.
+-->
+
+**Name**: TODO
+
+**Package**: `@stencil-community/<TODO>`
+
+### Description
+
+<!--
+Describe the platform.
```

Comment:
Nit - I think this word makes a little more sense in the Capacitor sense, but not as much in Stencil. How about:

```suggestion
Describe the package - be sure to include what it would do, its audience, and how it differentiates itself from any existing packages.  
```